### ISSUE TITLE
Add Search to Server installation Page

### ISSUE THREAD

**Issue by TheHomecraft** [BlueSquare23/web-lgsm#27]
**Is your feature request related to a problem? Please describe.**
It is time-consuming to scroll through the list of available servers to find the one I want to install.

**Describe the solution you'd like**
Add a search bar to the top of the page to make the installation process easier for the user.

**Describe alternatives you've considered**
Installation of instances via Steam ID.

**Comment by BlueSquare23** [comment]
Yeah this is a good idea. I've added it to the project's [Todos.txt](https://github.com/BlueSquare23/web-lgsm/blob/dev-1.8.0/Todos.md#v180-pt-4-the-rest) on the v1.8 dev branch.

Definitely needed cause I'm always hitting Ctrl + F on that page even myself, so yeah might as well just build it into the page. Thanks for the suggestion!

As you can see if you look at that Todos.txt file, I've got a lot of stuff for the v1.8 release to get wrapped up first. Probably going to take me another couple months of work on it. But its on the list! Will update the issue when the newest release is out.

**Comment by BlueSquare23** [comment]
Still probably another month or so until v1.8 is released. But just wanted to update this issue to say I got this feature finished. Just a little search box and some javascript to do the needful!

Commit: https://github.com/BlueSquare23/web-lgsm/commit/e6b4c7b93abf526c5620dbd5e5f662ce7481c901
 
 Will update and close this issue fully when v1.8 is released!

**Comment by BlueSquare23** [comment]
This is now done! Newest version 1.8.0 of the web-lgsm is out now and the install page now includes a working search bar!

![Image](https://github.com/user-attachments/assets/9b009a96-f1ce-4b4e-b05d-1765627236cc)
